### ISSUE TITLE
copytranslations() fails if there is an unexpected http error on a GET or POST call

### ISSUE THREAD

**Issue by sikaiser** [GreenpeaceSchweiz/weblatetools#3]

**Comment by sikaiser** [comment]
```
drupal-core 
>> getting -  Success: (200) OK 
>> writing -  Client error: (413) Request Entity Too Large 
Error in curl::curl_fetch_memory(url, handle = handle) : 
  HTTP/2 stream 0 was not closed cleanly: PROTOCOL_ERROR (err 1)
Calls: copyTranslations ... request_fetch -> request_fetch.write_memory -> <Anonymous>
In addition: Warning message:
In split.default(x = seq_len(nrow(x)), f = f, drop = drop, ...) :
  data length is not a multiple of split variable
Execution halted
Error: Process completed with exit code 1.
```

**Comment by sikaiser** [comment]
As far as I can tell, this is caused by an incorrect implementation of HTTP 2. Likely server-side.

In that case it should now be solved by 5ca20f32f546d1b639c38d6315b4863025f45361. However, hard to reproduce, since error doesn't always happen.
